### ISSUE TITLE
Direct build may be broken

### ISSUE THREAD

**Issue by luxzg** [sirskunkalot/PlanBuild#36]
Hi, to be clear from start I am NOT 100% sure but - direct build option (left CTRL) seems to be broken.

With today's patch the selection tool works again (select all connected option) so I was able to make a blueprint, and I am able to place that blueprint (and could use totem to build but it's huge, 10k wood 1k iron), but I wanted to check it out with direct build, and it just - did nothing. Checking the blueprint with crystal and fly mode looks like blueprint itself is ok.

Could someone else check this tool option and confirm? PlanBuild 0.9.3, released today.

Thanks!

P.s. and thanks for fixing select tool :)

**Comment by sirskunkalot** [comment]
There was a change regarding the direct build with one of the last patches. There is a new config for default build option now, maybe you actually did build a planned version using the modifier key? Gonna try to reproduce this now, though.

**Comment by sirskunkalot** [comment]
Another possibility: Direct build is by default only enabled for admins and has to be enabled for non-admins on a server via config.

**Comment by luxzg** [comment]
I will try deleting old config and let it recreate new one, maybe some conflict there or something. It's on my singleplayer instance, so I should be admin (?). I'll report tomorrow when I get back to PC

**Comment by sirskunkalot** [comment]
Yes, in local play you are always admin.

**Comment by luxzg** [comment]
Ok, so I checked. I deleted old config, made no change. Made change using config manager for few settings, then noticed I need to logout/login to apply them. But enabling direct build and making logout/load game did not enable direct build via Left Ctrl.

Now, finally I made Direct the default build mode via config, then made logout, reload save, and was able to place a direct build  BUT! Modifier key still did not work, so I lost ability to place planned build 

It's ok, I don't do that all so often, I can change config according to my current needs and reload the saved game. But it still seems weird.

Config only mentions Left Control as delete modifier key, not otherwise. So I'm going by the onscreen tips. When in default config, it says L-Shift for camera distance and L-Ctrl for direct build, and then x/y/z modifiers. Left Shift for camera works, so do all the x/y/z modifiers. So I don't think it's keybindig conflict of some kind.

Anyway, the take is that only the L-Ctrl + L-Mouse combo does not work (for me), while Direct and Planned builds do work when being placed just with L-Mouse.
